controllers/constant/odlm.go

```diff
@@ -356,6 +446,96 @@ spec:
     spec:
       grafana: {}
       operandRequest: {}
+  - name: cloud-native-postgresql
```

Comment:
I don't think we need to cover the SaaS scenario for EDB.
@horis233 Could you please confirm this?

Reply:
No, we don't need EDB in the SaaS mode